OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -243,6 +253,14 @@ public void Activate(Order order)
 			if (power == null)
 				return;
 
+			if (!HasSufficientFunds(power))
+				return;
```

Comment:
Use TakeCash() instead. it will return a bool wether that was successfull. return on false. It will also automaticaly play the "InsufficientFundsNotification", so you can remove the whole "HasSufficientFunds" method.